repositories.yaml

```diff
@@ -964,6 +964,27 @@ applications:
         additional_dependencies:
           - glean-core
 
+  - app_name: mozillavpn_cirrus
+    canonical_app_name: Mozilla VPN Cirrus Sidecar (?)
```

Comment:
It was in the bug, so I left it in there. I don't know what Bea's intent was here

Reply:
I think it might have indicated some lack of confidence in the naming? The cannonical name can easily be changed later, if needed, so let's just drop the (?)

Reply:
Update: We are getting clarification on the intent of this

Reply:
edited to take out `(?)`